e2e/nomostest/clusters/kind.go

```diff
@@ -94,13 +96,19 @@ func (c *KindCluster) Create() error {
 	if err != nil {
 		return err
 	}
-	err = createKindCluster(c.provider, c.Name, c.KubeConfigPath, version)
-	if err == nil {
-		c.creationSuccessful = true
-	} else {
+	tg := taskgroup.New()
+	tg.Go(func() error {
+		return createKindCluster(c.provider, c.Name, c.KubeConfigPath, version)
+	})
+	tg.Go(func() error {
+		return pullImages()
+	})
+	if err := tg.Wait(); err != nil {
 		c.creationSuccessful = false
```

Comment:
doesn't it default to false? why do we need to set it?

Reply:
removed